### PR TITLE
Remove the plus button for adding new items to the list.

### PR DESCRIPTION
**Pull request by p0psicles**
Items are now saved onChange event.

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: themes-default/slim/views/vue-components/select-list-ui.mako [hunk 1]
```diff
@@ -95,12 +87,15 @@ Vue.component('select-list', {
...
     },
     mounted() {
-        this.lock = true;
-        this.editItems = this.sanitize(this.listItems);
-        this.$nextTick(() => this.lock = false);
-        this.csv = this.editItems.map(x => x.value).join(', ');
+        this.constructList();
     },
     methods: {
+        constructList: function() {
```
Comment (p0psicles) [inline themes-default/slim/views/vue-components/select-list-ui.mako]: @sharkykh this got in accidentally as I was testing your fix, can remove later.

File: themes-default/slim/views/vue-components/select-list-ui.mako [hunk 2]
```diff
@@ -174,6 +168,12 @@ Vue.component('select-list', {
         },
         csv() {
             this.syncValues();
+        },
+        listItems: {
+            handler: function() {
+                this.constructList();
+            },
+            deep: true
```
Comment (sharkykh) [inline themes-default/slim/views/vue-components/select-list-ui.mako]: I don't think you even need `deep: true`. (it works without)
You're sending it a list of strings, or a string, so in any case no objects are involved.

**Comment by sharkykh** [comment]
My idea for the same issue:

**Comment by p0psicles** [comment]
Yeah have to say thats also pretty nice.

**Pull request closed without merging**